docs/developers/stored-requests.md

```diff
@@ -0,0 +1,138 @@
+# Stored Requests
+
+This document gives a technical overview of the Stored Impressions feature.
```

Comment:
Hey @dbemiller I think you meant 'Stored Requests'?  This appears to be the only mention of 'Stored Impressions'.